src/index.html

```diff
@@ -5,8 +5,425 @@
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="content">
+      <div
+      class="card"
+      data-qa="card"
+      >
+        <div class="card-image">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27-image"
+            class="card-image__model"
+          >
+
+          <h2 class="card-image__info">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </h2>
+          <h3 class="card-image__code">
+            Product code: 195434
+          </h3>
+
+
+          <div class="star stars--4">
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+
+            <section>
+              <aside class="star__reviews">
+                Reviews: 5
+              </aside>
+            </section>
+          </div>
+
+          <p class="card-image__text">
+            Price:
+            <span class="card-image__cost">$2,199</span>
+          </p>
+
+          <a
+            href="#"
+            data-qa="card-hover"
+            class="button"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-image">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27-image"
+            class="card-image__model"
+          >
+
+          <h2 class="card-image__info">
+            APPLE A1419 iMac 27" Retina
+            <br>
+            5K Monoblock (MNED2UA/A)
+          </h2>
+          <h3 class="card-image__code">
+            Product code: 195434
+          </h3>
+
+          <div class="star stars--4">
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+            <div class="star__stars"></div>
+
+            <div>
+              <div class="star__reviews">
+                Reviews: 5
+              </div>
```

Comment:
If i'm trying to use <p> instead of div inside the stars div , its not working. But even if i'm trying to create new div for whole star + reviews text with flex, it's not woking the same way , stars dont want to margin them selfs + cause of <p> tag more space is created (all elements is shifted down a bit) and not sure how to fix it properly , I'm confused